CHANGELOG.rst

```diff
@@ -5,23 +5,96 @@ Changelog
 0.17.0 (2024-05-??)
 ===================
 
-Refactor/rewrite release.
+This release is a big rewrite and refactor of the library internals.
 
-The custom views and backend have been rewritten to be more configurable out of the box,
-without needing to write much code in your own project. We've incorporated our
-experiences from the Open Forms and Open Inwoner projects in this rewrite.
+💥 There are a number of breaking changes, please review the notes further down.
```

Comment:
this is the magic of any version pre 1.0 :grimacing: every minor version in 0.x.y may technically contain breaking changes

I'm thinking that once we have the models streamlined (different instances rather than different models for different configurations), we can slap a 1.0 version on it.

Reply:
Ah I see, didn't know that :p